app/tests/spec/lib/fxa-client.js

```diff
@@ -64,19 +64,42 @@ function (chai, $, sinon, p, testHelpers, Session, FxaClientWrapper,
     });
 
     describe('signUp', function () {
-      it('signUp signs up a user with email/password', function () {
+      it('Sync signUp signs up a user with email/password and returns keys', function () {
         sinon.stub(realClient, 'signUp', function () {
-          return p({});
+          return p({
+            unwrapBKey: 'unwrapBKey',
+            keyFetchToken: 'keyFetchToken'
+          });
         });
 
         return client.signUp(email, password, relier)
-          .then(function () {
+          .then(function (sessionData) {
             assert.isTrue(realClient.signUp.calledWith(trim(email), password, {
               keys: true,
               service: SERVICE,
               redirectTo: REDIRECT_TO,
               resume: expectedResumeToken
             }));
+
+            assert.equal(sessionData.unwrapBKey, 'unwrapBKey');
+            assert.equal(sessionData.keyFetchToken, 'keyFetchToken');
```

Comment:
I am checking to ensure the keys from the server [are added to the updatedSessionData](https://github.com/mozilla/fxa-content-server/pull/2005/files#diff-341971be75d35dbd724949cda527a0efR119) when expected. I can add further assertions in the test below. Perhaps the [guard](https://github.com/mozilla/fxa-content-server/pull/2005/files#diff-341971be75d35dbd724949cda527a0efR118) is not necessary and we just return the keys if they are in the server returned accountData.